### ISSUE TITLE
Kinetic Fist (And possibly Acid element for Earth Kineticist?)

### ISSUE THREAD

**Issue by FrijDom** [Vek17/TabletopTweaks-Core#405]
**Is your feature request related to a problem? Please describe.**
I feel the game is lacking in ways of dealing elemental damage besides casting, especially acid, though the others suffer much the same. I feel Kinetic Blade limits you from feeling like you're actually hitting anything with it, and casting to get all your elemental damage gets boring after the first two playthroughs with a caster. I want the option to beat people up with acid-wrapped fists without having to spend my time trying to limit my Elemental Fist uses. I don't care about power. I want to punch them in the face with acid.

**Describe the solution you'd like**
As it says on the tin, both acid damage and Kinetic Fist are missing from the game. I'd like them added, and preferably compatible with one another.

**Describe alternatives you've considered**
I'll accept one or the other, though I feel they are disconnected enough that there shouldn't be a reason you can't add both (given time as you need, of course).

**Additional context**

**Comment by Shodan14** [comment]
There's always the [Elemental Ascetic archetype](https://www.d20pfsrd.com/alternative-rule-systems/occult-adventures/occult-classes/kineticist/archetypes/paizo-llc-kineticist-archetypes/elemental-ascetic-kineticist-archetype/)..

**Comment by FrijDom** [comment]
Yeah, but... Last I checked, that archetype isn't in the game. I'd be glad to see it added alongside the infusion that it's based around.

**Comment by Shodan14** [comment]
I meant that it would make sense to add that to make a kinetic fist build really work.

**Comment by FrijDom** [comment]
I absolutely agree! If that's something you'd like to do, I'd love to see that added.
